neofs-testlib/neofs_testlib/env/templates/neofs_env_config.yaml

```diff
@@ -1,49 +1,44 @@
 binaries:
     neofs_cli:
         repo: 'nspcc-dev/neofs-node'
-        version: 'v0.39.2'
+        version: 'v0.42.0'
         file: 'neofs-cli-amd64'
```

Comment:
In the latest releases the file name has been changed and requires to be updated as well:
* `neofs-cli-amd64` -> `neofs-cli-linux-amd64`
* `neofs-adm-amd64` -> `neofs-adm-linux-amd64`
* `neofs-ir-amd64` -> `neofs-ir-linux-amd64`
* `neofs-lens-amd64` -> `neofs-lens-linux-amd64`
* `neofs-node-amd64` -> `neofs-node-linux-amd64`

Reply:
fixed